ansible_base/jwt_consumer/common/auth.py

```diff
@@ -135,6 +133,12 @@ def parse_jwt_token(self, request):
                         defaults=user_defaults,
                     )
 
+                    resource = Resource.get_resource_for_object(self.user)
+
+                    resource.ansible_id = self.token['sub']
+                    resource.service_id = self.token['service_id']
+                    resource.save()
```

Comment:
Do you want to specify `update_fields` on this save? Just thinking we know exactly what fields we need to change here so if there is any kind of race condition with the sync process this might help.
```suggestion
                    resource.save(update_fields=['ansible_id', 'service_id'])
```